autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/sync/CredentialsSyncDataPersister.kt

```diff
@@ -63,7 +65,9 @@ class CredentialsSyncDataPersister @Inject constructor(
     }
 
     override fun onError(error: SyncErrorResponse) {
-        // TODO: implement
+        if (error.type == CREDENTIALS) {
```

Comment:
to follow-up on our conversation, have you created a task to move this out of the persister?